ydb/core/mind/hive/hive.h

```diff
@@ -105,6 +106,12 @@ EResourceToBalance ToResourceToBalance(NMetrics::EResource resource);
 
 struct ISubActor {
     virtual void Cleanup() = 0;
+
+    virtual TString GetDescription() const {
```

Comment:
I think it would be better to ether make it pure virtual or wite some meaningful implementation (for example using TypeName(*this) to obtain name of the implementation class)